### ISSUE TITLE
User Requirements Review

### ISSUE THREAD

**Issue by ykaya932** [bounswe/bounswe2020group8#28]
User Requirements will be reviewed and completed and glossary will be filled with unclear definitions according to the feedback in piazza. Deadline Sunday 23.59

[Here](https://github.com/bounswe/bounswe2020group8/wiki/Requirements) is the link.

**Comment by afraarslan** [comment]
There is something wrong with the headlines of user requirements.  The numbers are not matched with the ordered list. Can you please correct them?

**Comment by nursimacelik** [comment]
Numbering of user requirements are corrected.

**Comment by nursimacelik** [comment]
As the task is not completed by the deadline(1.03.2020) or the customer meeting (5.03.2020), reassignment is needed.
New deadline is 8.03.2020.

**Comment by nursimacelik** [comment]
We reviewed the user requirements considering the [customer meeting 1](https://github.com/bounswe/bounswe2020group8/wiki/Customer-Meeting-%231)

Closing the issue. If a problem arises, it can be reopen.

**Comment by nursimacelik** [comment]
There are [new feedbacks](https://piazza.com/class/k5z7hzm7ncm1m3?cid=69).

I updated the document (easy ones):
1- Removed that sentence.
3 - Removed the Requirement 1.1.3.2: "Vendors shall be able to sell the same product."
4 - Put previous 1.1.4.2 and 1.1.4.3 under 1.1.4.1.
5 - Changed "should" to "shall" in 1.1.4.3 and 1.1.4.4 . (previously 1.1.4.5 and 1.1.4.6)

Problems:
10 - I defined campaign in the glossary but there is no req. about vendors making a campaign.
12 - A customer may want to ask for product details before buying a product. Should we ignore this case?
13 - I am indecisive about threshold. Would one week be long?
